helm/helm.bzl

```diff
@@ -107,7 +107,7 @@ helm_push_ = rule(
         ),
         "_push_script_template": attr.label(
             allow_single_file = True,
-            default = "//tools/build/helm:push.sh",
+            default = "//helm:push.sh",
```

Comment:
When used in another repository this would look for the file at `helm/push.sh` from the workspace root. To make it look in this repository we should use `@com_github_airyhq_bazel_tools//helm:push.sh`

Reply:
Ah, you are right, thanks :+1: